dask/array/creation.py

```diff
@@ -12,8 +12,162 @@
 from ..base import tokenize
 from ..utils import ignoring
 from . import chunk
-from .core import Array, normalize_chunks, stack, concatenate
-from .wrap import empty
+from .core import Array, asarray, normalize_chunks, stack, concatenate
+from .wrap import empty, ones, zeros, full
+
+
+def empty_like(a, dtype=None, chunks=None):
```

Comment:
These `*_like` functions in NumPy also take `order` and `subok` parameters. Though these didn't make much sense for Dask Array's IMHO as C-order arrays are used exclusively and am not aware of any subclass handling logic related to Dask Array's.

Reply:
Agreed